.gitignore

```diff
@@ -41,7 +41,7 @@ src/main/c/*/Makefile
 src/main/c/*/extconf.h
 src/main/c/*/mkmf.log
 src/main/c/etc/constdefs.h
-src/main/c/spawn-helper/spawn-helper
+src/main/c/spawn-helper/spawn-helper*
```

Comment:
Which file do you see in `git status`?
Maybe a `.dSYM` of macOS or so?

Reply:
Yes I see `src/main/c/spawn-helper/spawn-helper.dSYM/Contents/Resources/DWARF/spawn-helper` does nobody else see it?

Reply:
Not on Linux for sure, what about you @norswap / @bjfish?

Reply:
I don't have it on Catalina. But I have been building mostly `jvm` the past two weeks, so it's possible it is missing because it only appears in a particular config?

Reply:
We should probably exclude `src/main/c/spawn-helper/spawn-helper.dSYM` explicitly, because e.g. `src/main/c/spawn-helper/spawn-helper.c` is actually a tracked file. `git` seems to handle that OK, but it might lead to surprising behavior if a file is both ignored and checked in the repo.

Maybe we should ignore `*.dSYM` globally for macOS actually?

Reply:
> Maybe we should ignore `*.dSYM` globally for macOS actually?

That'd be fine.

Reply:
OK, I'll add that.